src/scripts/main.js

```diff
@@ -22,7 +22,19 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
-}
+  if (Object.keys(data).length === 0) {
+    return false;
+  } else {
+    for (const [key, value] of Object.entries(data)) {
```

Comment:
You can use 'for in' statement instead
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in